### PR TITLE
Add reference content in lieu of procedure that was removed by #29454

### PR DESCRIPTION
**Pull request by rolfedh**
Restore include statement that I commented out in https://github.com/quarkusio/quarkus/pull/29454

### CODE REVIEW

**Comment by gsmet** [comment]
Can you please make sure the title of the PR is descriptive? We don't want such cryptic titles.

File: docs/src/main/asciidoc/security-built-in-authentication-support-concept.adoc [hunk 1]
```diff
@@ -10,11 +10,11 @@ include::_attributes.adoc[]
 
 The following section describes the Quarkus built-in authentication mechanisms for HTTP based FORM, BASIC, and Mutual TLS authentication. Proactive authentication is also described.
 
-include::security-basic-authentication-concept.adoc[leveloffset=+1]
+== Basic Authentication
```
Comment (michelle-purcell) [inline docs/src/main/asciidoc/security-built-in-authentication-support-concept.adoc:13]: @rolfedh - I'm just wondering whether this section belongs in this adoc or in `security-basic-authentication-concept.adoc`?

File: docs/src/main/asciidoc/security-overview-concept.adoc [hunk 2]
```diff
@@ -9,7 +9,7 @@ Quarkus Security is a framework that provides the architecture, multiple authent
 
 Before you start building security into your Quarkus applications, review the overview information to learn about the Quarkus Security architecture and the different authentication and authorization mechanisms that Quarkus supports.
 
-To get started with security in Quarkus, we recommend that you first combine the Quarkus built-in xref:security-basic-authentication-concept.adoc[Basic HTTP authentication] with the JPA identity provider to enable role-based access control (RBAC). 
+To get started with security in Quarkus, we recommend that you first combine the Quarkus built-in xref:security-basic-authentication-concept.adoc[Basic HTTP authentication] with the JPA identity provider to enable role-based access control (RBAC).
```
Comment (michelle-purcell) [inline docs/src/main/asciidoc/security-overview-concept.adoc:12]: This is probably outside the scope of the PR but should the `the JPA provider` be `a JPA provider` or `your JPA provider`?

**Comment by rolfedh** [comment]
Closing this PR as duplicate of @michelle-purcell 's work.

**Pull request closed without merging**